.ci/atime/tests.R

```diff
@@ -22,6 +22,38 @@ for (extra.arg in extra.args.6107){
   extra.test.list[[sprintf("fread(%s) improved in #6107", extra.arg)]] <- this.test
 }
 
+## Test cases adapted from https://github.com/Rdatatable/data.table/pull/7264#issuecomment-3228438559 which was adapted from R code added to NEWS.md file in that PR.
+roll.7264.list <- atime::atime_grid(
+  "frollapply improved"=data.table::frollapply(value, len_window, max, adaptive=TRUE, align="left"),
+  "frollmax created"={
+    tryCatch({
+      data.table::frollmax(value, len_window, adaptive=TRUE, align="left", has.nf=FALSE)
+    }, error=function(e){
+      data.table::frollapply(value, len_window, max, adaptive=TRUE, align="left")
+    })
+  },
+  "frollmean improved"=data.table::frollmean(value, len_window, adaptive=TRUE, align="right"))
```

Comment:
Comment above this atime test cases "Test cases adapted from" seems to be not correct for this one.
froll mean, align=right
What's the purpose of this test in context of this PR?

Argument align=right is the default one so could be easily skipped here